fusor-ember-cli/app/templates/add-node-registration.hbs

```diff
@@ -0,0 +1,74 @@
+{{#em-modal configName="bs" id="addNodeModal" closeIfClickedOutside=handleOutsideClick open-if=openAddNodeRegistrationModal close-if=closeAddNodeRegistrationModal}}
+  {{#em-modal-title}}
+    {{#em-modal-toggler class="close"}}
+      <span aria-hidden="true">×</span><span class="sr-only">Close</span>
+    {{/em-modal-toggler}}
+    <h4 class="modal-title">Add Node(s) to {{nodeInfo.address}}</h4>
+  {{/em-modal-title}}
+
+  {{#em-modal-body}}
+    <div class="row">
+      <div class="col-xs-12">
+        <p>Enter information for additional nodes that should be registered to {{nodeInfo.address}}</p>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-xs-offset-1 col-xs-11">
+        <form class="form-horizontal add-node-form">
+
+          {{#base-f cssId="addDriverInput"
```

Comment:
Doesn't look like cssId is defined to do anything on a `base-f`. Are these intended to assist QE?